repositories/repositories.bzl

```diff
@@ -225,11 +225,12 @@ def ros2_repositories():
         urls = ["https://github.com/ros2/rmw_cyclonedds/archive/0.7.6.tar.gz"],
     )
 
+    # ros2-devel branch
     maybe(
         http_archive,
-        name = "ros2_unique_identifier_msgs",
```

Comment:
This looks a bit off :) Please create a new maybe+http_archive declaration instead of replacing existing `ros2_unique_identifier_msgs`.